app/src/main/java/edu/uc/kovaciad/slicetracker/MainActivity.kt

```diff
@@ -7,11 +7,16 @@ import androidx.room.Room
 import edu.uc.kovaciad.slicetracker.dao.SliceDatabase
 import edu.uc.kovaciad.slicetracker.dto.Brand
 
+import kotlinx.android.synthetic.main.activity_main.*
 
+/**
+ * This main activity is just a container for our fragments,
+ * where the real action is.
```

Comment:
JavaDoc/KDoc above a class is a good idea.  But... it should explain more than the user can read from the code itself.